src/ring_theory/norm.lean

```diff
@@ -160,4 +161,81 @@ algebra.norm_eq_zero_iff_of_basis (basis.of_vector_space K L)
 
 end eq_zero_iff
 
+namespace intermediate_field.adjoin_simple
+
```

Comment:
We are already in the `algebra` namespace, and the lemma name `norm_eq_norm_adjoin` already contains `adjoin`, so I'd just stay in the same namespace:
```suggestion
```